### PR TITLE
Coursier.Coursier version 2.1.4

### PR DESCRIPTION
**Pull request by superusercode**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109484&drop=dogfoodAlpha

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
VC Redist missing:
image

**Comment by ItzLevvie** [comment]
[Policy] Dependencies

waiting until dependency support is out of experimental features and is enabled in pipelines

**Comment by stephengillie** [comment]
Manual Validation ended with:
> PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Coursier.Coursier__DefaultSource\cs-x86_64-pc-win32.exe       ________ ___   / /  ___     / __/ __// _ | / /  / _ |   __\ \/ /__/ __ |/ /__/ __ |  /____/\___/_/ |_/____/_/ | |                           |/  Checking if a JVM is installed   No JVM found, should we try to install one? [Y/n]

(Automated response - build 487.)

File: manifests/c/Coursier/Coursier/2.1.4/Coursier.Coursier.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: cs-x86_64-pc-win32.exe
+    PortableCommandAlias: cs
+  InstallerSha256: 13525D0D4636818FC366E3647C857260A868DEF8FF7E944907ED0DE983E54610
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
Comment (stephengillie) [inline manifests/c/Coursier/Coursier/2.1.4/Coursier.Coursier.installer.yaml:17]:
```suggestion
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
  - PackageIdentifier: Oracle.JavaRuntimeEnvironment
```
Hi @superusercode,

The error suggests that we should have a `JRE` as a dependency, but there are many implementation. We have plans for a way to specify multiple or any of a class of dependency, but for now we can omit this if you want.

**Pull request closed without merging**